src/Controllers/CopyPatrol.php

```diff
@@ -108,6 +108,10 @@ protected function handleGet() {
 				$records[$key]['editor_page_dead'] = false;
 				$records[$key]['editor_talk_dead'] = false;
 			}
+			if ( $record['status_user'] ) {
+				$records[$key]['reviewed_by_url'] = $this->getUserPage( $record['status_user'] );
+			}
+			$records[$key]['review_timestamp'] = $this->formatTimestamp( $record['review_timestamp'] );
```

Comment:
Well, for now anyway I figure since we're allowing unauthenticated users to perform reviews, we should at least record the timestamp. Once we require authentication we could probably get rid of the if statement altogether